src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -4431,5 +4431,54 @@ .maxstack  2
                 Row(6, TableIndex.MethodDef, EditAndContinueOperation.Default),
                 Row(7, TableIndex.MethodDef, EditAndContinueOperation.Default));
         }
+
+        [Fact]
+        public void TopLevelStatement_Closure()
+        {
+            var source0 = MarkedSource(@"
+<N:0>
```

Comment:
What syntax node does this denote? The CompilationUnit?

Reply:
Yes